src/exes/moc.ml

```diff
@@ -27,13 +27,21 @@ let link = ref true
 let interpret_ir = ref false
 let gen_source_map = ref false
 let explain_code = ref ""
+let stable_types = ref false
 
 let argspec = [
   "-c", Arg.Unit (set_mode Compile), " compile programs to WebAssembly";
   "-g", Arg.Set Flags.debug_info, " generate source-level debug information";
   "-r", Arg.Unit (set_mode Run), " interpret programs";
   "-i", Arg.Unit (set_mode Interact), " run interactive REPL (implies -r)";
   "--check", Arg.Unit (set_mode Check), " type-check only";
+  "--compatible",
```

Comment:
Perhaps rather call this `stable-compatible`? It does not check compatibility of the actor interface itself, so the name would be a bit misleading.

Reply:
Well, I was hoping we would extend --compatible to support other inputs as well, based on file extension.
*.wasm  compare both candid:service and motoko:stable-types, extracted from
*.did compare service (ignoring any service arguments)
*.sig compare sig
* (no extension) compare both *.did and *.sig
But I guess we could add the general option later.

Reply:
Oh yes, I'd like that. But as long as it doesn't do that perhaps better to not call it that.